tests/test_marc.py

```diff
@@ -85,7 +85,7 @@ def test_line_to_bin_unicode(self):
                 marc = marcs.next()
                 self.assertTrue(marc.author() == \
                                 u'ƀƁƂƃƄƅƆƇƈƉƊƋƌƍƎƏƐƑƒƓƔƕƖƘƙƚƛƜƝƞƟƠơ '\
-                                '1900-1980 Verfasser$0(DE-588)118536389$4aut',
+                                '1900-1980 Verfasser (DE-588)118536389 aut',
                                 "Line MARC title didn't match pymarc title")
```

Comment:
To keep this test in line with the message you could do:
```
 self.assertEquals(marc.title(), u'ΛΦϞЌЍЖ⁁⅀∰     ﬡﬢ－中英字典こんにちはß')
                                "Line MARC title didn't match pymarc title")
```
and avoid issues with the subfields completely..

Also I support your use of `assertEquals()` that you have in your other PR #20 for making the test assertions (and default comparison messages) clearer, although separating the test fixes from python 3 upgrade would be nice :)